StaticAnalyzer/views/android/manifest_analysis.py

```diff
@@ -236,48 +331,16 @@ def manifest_analysis(mfxml, man_data_dic):
         for application in applications:
 
             if application.getAttribute("android:debuggable") == "true":
-                ret_value = (
-                    ret_value + (
-                        '<tr><td>Debug Enabled For App <br>[android:debuggable=true]</td><td>'
-                        '<span class="label label-danger">high</span></td><td>Debugging was enabled'
-                        ' on the app which makes it easier for reverse engineers to hook a debugger'
-                        ' to it. This allows dumping a stack trace and accessing debugging helper '
-                        'classes.</td></tr>'
-                    )
-                )
+                ret_list.append(("a_debuggable",tuple(),tuple(),))
```

Comment:
I was thinking of doing a code refactoring. Yes these lists are faster than string concatenation.